applications/ShallowWaterApplication/custom_utilities/move_shallow_water_particle_utility.h

```diff
@@ -447,7 +447,7 @@ class MoveShallowWaterParticleUtility
     {
         KRATOS_TRY
         ModelPart::NodesContainerType::iterator inodebegin = rNodes.begin();
-        vector<unsigned int> node_partition;
+        std::vector<unsigned int> node_partition;
```

Comment:
also my advice would be to directly use the openmp for this.

look e.g. here for an example

https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/variable_utils.cpp#L178

to make it clear this is just a suggestion, NOT a request

Reply:
Sorry, I had forgotten that. I'm going to merge and close. my plans to the future is to use the implementation of pfem2App and to remove this duplicated code